package.json

```diff
@@ -51,6 +50,12 @@
   },
   "dependencies": {
     "aws-sdk": "~2.2.28",
+    "babel-cli": "^6.24.1",
```

Comment:
They're needed at runtime because we're using the transform runtime plugin. It's not really that big of a deal, though. Package size isn't increased by much.